src/battle_script_commands.c

```diff
@@ -16559,3 +16563,18 @@ void BS_ItemRestorePP(void) {
     PREPARE_SPECIES_BUFFER(gBattleTextBuff1, GetMonData(mon, MON_DATA_SPECIES));
     gBattlescriptCurrInstr = cmd->nextInstr;
 }
+
+void BS_TryRevertWeatherForm(void)
+{
+    NATIVE_ARGS();
+    u16 ability = gBattleMons[gBattlerTarget].ability;
+    if (B_WEATHER_FORM_SUPPRESS >= GEN_5
```

Comment:
`B_WEATHER_FORM_SUPPRESS` is written as a generation config and so I don't think I can just write `#ifdef B_WEATHER_FORM_SUPPRESS` since it's always defined right now.